ampd/src/event_processor.rs

```diff
@@ -164,11 +229,14 @@ where
         Ok(msgs) => {
             if is_avalanche_evm_handler(handler_label) {
                 for (i, msg) in msgs.iter().enumerate() {
+                    let deserialized_values = show_protobuf_structure(&msg.value).unwrap();
```

Comment:
I don't think we should panic or error here, because if there is a bug or unexpected error on this function this will break event handling. Maybe log the error message instead?